CRM/Contact/BAO/Query.php

```diff
@@ -4336,6 +4336,8 @@ public static function getQuery($params = NULL, $returnProperties = NULL, $count
    * @param string $apiEntity
    *   The api entity being called.
    *   This sort-of duplicates $mode in a confusing way. Probably not by design.
+   * @param bool $useGroupBy
+   *   Should the query be grouped by contact id?
```

Comment:
Oh, okay. I've been decorating bool values with questions for years now. Don't you know this from form checkboxes. I'm asking the user – not mysqlf ;)